packages/api/src/orbitAPI.ts

```diff
@@ -93,13 +68,71 @@ export type ValidatableSpec = {
       response?: ResponseList<"taskData", PromptID, Prompt>;
     };
     PATCH?: {
+      contentType: JSONContentType;
       body: { id: PromptID; data: Prompt }[];
       response?: null;
     };
   };
+
+  "/attachments"?: {
+    /**
+     * encode with multipart/form-data, with the file in part named "file"
+     * make sure to include Content-Type heading for your attachment
+     * returns application/json encoded ResponseObject<"attachmentIDReference", AttachmentID, AttachmentIDReference>
+     */
+    POST?: {
+      // NOTE: Content-type must use regex to be validated since additional data is usually
+      // included alongside the contentType (i.e. the form-data length)
```

Comment:
Ah, juicy! Makes sense. It took me a moment to understand this comment: perhaps "…since additional data, like the form-data length, is usually appended to the MIME type" would be clearer?

Reply:
Occurs to me: I suppose the pattern should actually be `^multipart/form-data` (i.e. enforcing the string as prefix), right?